packages/frontend/src/pages/campaigns/filters/oracles/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { useOracleTemplates } from "@carrot-kpi/react";
+import { CARROT_DEFAULT_ORACLE_TEMPLATE } from "../utils";
+import { Template } from "@carrot-kpi/sdk";
+import { CheckboxesFilter } from "../checkboxes-filter";
+
+export const OraclesFilter = ({
+    selectedOracles,
+    setSelectedOracles,
+}: {
+    selectedOracles: Set<number>;
+    setSelectedOracles: (newSelectedOracles: Set<number>) => void;
+}) => {
+    const { loading, templates } = useOracleTemplates();
+
+    const defaultChecked = (item: Template) =>
```

Comment:
Not sure I understand this and why it is a function.

Reply:
this function is to tell the `CheckboxesFilter` component what are the checkboxes that we want to be checked by default.

Reply:
We do not need this function because:

https://github.com/carrot-kpi/v1-monorepo/pull/206/files#r1152053453